packages/export-tools/src/find-unused-exports.mjs

```diff
@@ -0,0 +1,143 @@
+import { Project } from 'ts-morph';
+import { writeFile } from './utils.mjs';
+
+// 1. Locate ui-components, ui-premium, and labkey home via env-vars (consult labkey build code)
+// 2. Find all exports from ui-components, ui-premium (use ts-morph)
+// 3. Find all imports of ui-components and ui-premium in known locations based on env-vars (use ts-morph)
+// 4. Diff the exports and imports, output the diffs (one file per package)
+// 5. Consider using ts-morph to modify the AST and remove the unused exports (consult ts-morph docs)
+//     - Only do this when --write flag is present, don't output diff files when write is true
+//     - Bonus points: consider using ts-morph to export everything in sorted order.
+
+// S-1, S-999
+// S-1, S-2
+
+if (process.env.LABKEY_HOME === undefined) {
+    throw new Error('Missing env var LABKEY_HOME');
+}
+
+if (process.env.LABKEY_UI_COMPONENTS_HOME === undefined) {
+    throw new Error('Missing env var LABKEY_UI_COMPONENTS_HOME');
+}
+
+if (process.env.LABKEY_UI_PREMIUM_HOME === undefined) {
+    throw new Error('Missing env var LABKEY_UI_PREMIUM_HOME');
+}
+
+const components = `${process.env.LABKEY_UI_COMPONENTS_HOME}packages/components/src`;
+const premium = `${process.env.LABKEY_UI_PREMIUM_HOME}src`;
+let ehrComponents;
+const modules = `${process.env.LABKEY_HOME}server/modules`
```

Comment:
The paths I had set locally for LABKEY_UI_COMPONENTS_HOME, LABKEY_UI_PREMIUM_HOME, and LABKEY_HOME didn't have trailing slashes "/" at the end of the path so the first time I ran findUnused it came up with 0 lines written. Not sure if we care or need to support that case. When I updated my paths to include the /, it worked as expected.

Reply:
I'll update it to append a trailing slash if not present